rfcs/055-prismic-api/articles.md

```diff
@@ -0,0 +1,114 @@
+# Prismic API: articles endpoint
+
+## Example response
+
+```
+GET /articles/{id}
+{
+   {
+    "type": "Article",
+    "id": "XAEEIhQAACsA4eEl",
+    "format": {
+        "type": "Format",
+        "id": "W7TfJRAAAJ1D0eLK",
+        "name": "article"
```

Comment:
What is this called in the Prismic API? Let's not confuse labels and human readable slugs here.

Reply:
you're right, this should be an `Article`